src/assets/SideNav.css

```diff
@@ -30,7 +30,7 @@
   
   /* Color/shape of close button cross */
   .bm-cross {
-	background: var(--bg-light-900);
```

Comment:
Try to change it on any other value from src/variables.css. There are some accent values for both dark and light modes

Reply:
okay I will change it,

Reply:
This was only applied to the SideNav.  I forgot to mention this is in all of the panels in dark mode and refers to the header of each panel component.

Reply:
okay